draft-ietf-rats-corim.md

```diff
@@ -1546,25 +1546,26 @@ These inputs are added with the Reference Value Provider's authority.
 ### **Phase 4**: Endorsed Values Augmentation
 
 During Phase 4, Endorsed Values inputs containing conditions that describe expected Attester state are compared.
-If the comparison is satisfied, then additional Claims about the Attester are added.
+If the comparison is satisfied, then additional Claims about the Attester are added to the Accepted Claim Set.
```

Comment:
```suggestion
If the comparison is satisfied, then additional Claims about the Attester are added to the Actual-State Claim Set.
```

Reply:
Bulk of the document states ACS as Accepted Claims Set.  We need to agree on a common term and replace the entire document for consistency!

Reply:
Indeed "actual" is more philosophical. We're only accepting an authority's attestation of actual state. Ceci n'est pas une pipe.

Reply:
I'm OK with "Actual-state Claims Set" but it should be applied consistently. I prefer using ACS everywhere and define ACS in one place. Then it's easy to make a global change.